server/commands.lua

```diff
@@ -13,6 +13,23 @@ lib.addCommand('givekeys', {
     TriggerClientEvent('qb-vehiclekeys:client:GiveKeys', source, args.id)
 end)
 
+local function getVehicleKeysParams(source, args)
+    local playerId = args.target
+    local plate = args.plate
+
+    if not playerId then
+        playerId = source
+    end
```

Comment:
```suggestion
    local playerId = args.target or source
    local plate = args.plate
```
optional; just a succinct syntax alternative